src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -653,6 +655,44 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
         }
     };
 
+    const positionLookUp = (node = null) => {
+        let defaultPositionX = 100;
+        let defaultPositionY = 100;
+        const areaDistanceX = 250;
+        const areaDistanceY = 80;
+        const maxWidth = 800;
+        const defaultPosition = 100;
+        const minWidth = 300;
+        const minHeight = 100;
+        const lookUpElements = node ? [...elements, ...node] : [...elements];
```

Comment:
Is `node` the best name for this param? It looks like it's an array of elements?